src/renderer/components/dock/dock-store/dock.store.ts

```diff
@@ -224,6 +233,15 @@ export class DockStore implements DockStorageState {
     }), reactionOpts);
   }
 
+  onTabsNumberChange(callback: (evt: DockTabsNumberEvent) => void) {
+    return reaction(() => this.tabsNumber, ((currentTabsNumber, previousTabsNumber) => {
```

Comment:
This reaction is quite simple and I don't see any benefits of abstraction unless `tabsNumber` is `private` or `protected` field.

Reply:
Makes sense. Cleaned up.